packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -854,6 +855,12 @@ class SemanticsProperties extends DiagnosticableTree {
   /// This is mutually exclusive with [toggled].
   final bool? checked;
```

Comment:
What is this `checked` value for a tristate checkbox with value of `null`? If `checked` has a value of `null` in this scenario, I'd consider tweaking this field's comment to clarify this scenario.

Reply:
Its value should be `false`. I will add a comment.